pallets/sora2-trustless-bridge/common/src/merkle_proof.rs

```diff
@@ -0,0 +1,81 @@
+use ethabi::{encode, Token};
```

Comment:
it seems like its an old version of file in bridge-common, strange that this file is included there